great_expectations/rule_based_profiler/types/data_assistant_result.py

```diff
@@ -23,6 +26,38 @@ class DataAssistantResult(SerializableDictDot):
     ] = None  # Obtain "meta/details" using "meta = expectation_suite.meta".
     execution_time: Optional[float] = None  # Execution time (in seconds).
 
+    def plot(self, prescriptive: bool = False):
+        metrics: Dict[Dict, ParameterNode] = self.metrics
```

Comment:
@NathanFarmer Do we need `ParameterNode` here?  The reason for the question is that `metrics` in `DataAssistantResult` have the type of `Dict[Domain, Dict[str, Any]]`, which include access to both `attributed_values` and details.

Reply:
@alexsherstinsky according to my debugger this type hint is correct.

Reply:
@NathanFarmer We also do not want to use `Dict` along as a type hint (it would need to be something like `Dict[str, some_other_type]`.